### ISSUE TITLE
[EN] [Dungeon] [Dire Maul - Capital Gardens (West)] Missing Pylons

### ISSUE THREAD

**Issue by Tj20** [Atlantiss/BugTracker#1213]
**Expected behaviour:** [Immol'thar](http://www.wowwiki.com/Immol%27thar) should be in his prison and unaccessible to the group until they destroy some magical pylons guarded by [Mana Remnants](http://www.wowhead.com/npc=11483) and [Arcane Abberrations](http://www.wowhead.com/npc=11480).

![how it should look like](http://i.imgur.com/VeT6gJ6.png)

**Observed behaviour:** The mobs are there, but not the pylon models. Also the barrier around Immol'thar's prison is missing, so the group can kill him without killing the pylon guardians first.

![how it is](http://i.imgur.com/SAl1Knt.jpg)

/edit: I think I should title my reports according to the guide on the front page, not according to what labels there are available. Let me try that again with the title.

**Comment by Bildo** [comment]
fixed

**Comment by roirraWedorehT** [comment]
[https://github.com/Atlantiss/BugTracker/issues/6244](url)

Confirmed that the quest works fine, deactivate the pylons by standing directly underneath the base and it makes a sound and looks different when it's deactivated, plus the gear symbol no longer appears (unlike when you unsuccessfully deactivate by not standing in the right place). Also we noticed pylon #1 had a targetable/attackable "enemy" by the name of the pylon, and we happened to kill it before deactivating the pylon.

It might not have been necessary but we deactivated them in pylon number order, 1 through 5.
